PY-SDK/admin/instructions/supplement_init_market.py

```diff
@@ -0,0 +1,60 @@
+from ...public.src.pyaver.aver_client import AverClient
+from ...public.src.pyaver.market import AverMarket
+from ...public.src.pyaver.orderbook import Orderbook
+from typing import NamedTuple
+from solana.publickey import PublicKey
+from solana.keypair import Keypair
+from solana.system_program import SYS_PROGRAM_ID
+from anchorpy import Program, Context
+
+class SupplementInitMarketArgs(NamedTuple):
+  outcome_id: int
+  outcome_names: list[str]
+  event_capacity: int
+  nodes_capacity: int
+
+class SupplementInitMarketAccounts(NamedTuple):
+  payer: Keypair
+  market: PublicKey
+  market_authority: PublicKey
+
+async def supplement_init_market_tx(
+  aver_client: AverClient,
+  args: SupplementInitMarketArgs,
+  accs: SupplementInitMarketAccounts,
+  program_id: PublicKey = None
+):
+  if program_id:
+    program = await aver_client.get_program_from_program_id(program_id)
+  else:
+    program = aver_client.programs[0]
```

Comment:
here, we could, in theory actually use the program_id from the market (it's a bit OTT, but just a note for the future)